composer.json

```diff
@@ -21,6 +21,7 @@
         "doctrine/inflector": "^1.1",
         "dragonmantank/cron-expression": "^2.0",
         "erusev/parsedown": "^1.7",
+        "judahnator/json-manipulator": "^1.2",
```

Comment:
`illuminate/database` can also used standalone, you would also need to add it to `Illuminate/Database/composer.json`.

Reply:
Ah! Good catch! I appreciate the feedback and have updated the PR.